docs/how-to/how-to-set-sandbox-config-kata.md

```diff
@@ -47,6 +47,7 @@ There are several kinds of Kata configurations and they are listed below.
 | `io.katacontainers.config.hypervisor.block_device_driver` | string | the driver to be used for block device, valid values are `virtio-blk`, `virtio-scsi`, `nvdimm`|
 | `io.katacontainers.config.hypervisor.cpu_features` | `string` | Comma-separated list of CPU features to pass to the CPU (QEMU) |
 | `io.katacontainers.config.hypervisor.default_max_vcpus` | uint32| the maximum number of vCPUs allocated for the VM by the hypervisor |
+| `io.katacontainers.config.hypervisor.default_maxmemory` | uint32| the maximum memory assigned for a VM by the hypervisor in `MiB` |
```

Comment:
What is the use case? It's not really consistent with the commit message that talks about limiting hotplugging, not limiting the amount of memory the hypervisor takes.

I _think_ the idea is to ensure that you don't allocate beyond NUMA memory, but you should detail that a lot more in the commit message. What happens:
* If you add more memory than anticipated in the NUMA case?
* What does "hotplug become fully NUMA-aware" mean exactly? (What currently happens that is bad for Kata)

Reply:
Let me review the old commits. One issue with NUMA is that we cannot hot-plug memory.

Reply:
My understanding (limited, very limited), is that memory and vcpu hotplug is quite complicated with NUMA, if do-able at all.

This is not really related to the content of this commit, but I wonder if we can add a logic to ensure that NUMA only works in case `static_sandbox_mgmt` is set to true, and then we avoid any kind of unexpected surprises for casual users.

Last but not least, we need to make sure that users will have a good message, info mesage, about the NUMA state and why it wasn't enabled on cases where they'd expect it to be.

Reply:
I think we should fail in this scenario and give a proper error message. I do not want to enable static resource mgmt implicitly.

Reply:
I completely agree.

I also think a comment should be added in the config file saying something towards the lines of "if enabling numa support static_sandbox_resource_mgmt must also be enabled, as there's not hotplug support for NUMA"